filament/include/filament/RenderableManager.h

```diff
@@ -362,6 +382,11 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
             MaterialInstance const* materialInstance = nullptr;
             PrimitiveType type = PrimitiveType::TRIANGLES;
             uint16_t blendOrder = 0;
+            struct {
+                MorphTargetBuffer* buffer = nullptr;
+                size_t offset = 0;
+                size_t count = 0;
```

Comment:
I'm confused, I don't see where `offset` or `count` are getting read?

Reply:
Right now it's not used and need to implement support these values.